app/src/main/java/io/github/androidacademyglobal/ui/theme/Theme.kt

```diff
@@ -1,47 +1,92 @@
 package io.github.androidacademyglobal.ui.theme
 
+import androidx.compose.foundation.LocalIndication
 import androidx.compose.foundation.isSystemInDarkTheme
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.darkColors
-import androidx.compose.material.lightColors
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.material3.lightColorScheme
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.graphics.Color
 
-private val DarkColorPalette = darkColors(
-    primary = Purple200,
-    primaryVariant = Purple700,
-    secondary = Teal200
+private val AndroidAcademyDarkColorScheme = darkColorScheme(
+    primary = Blue80,
+    onPrimary = Blue20,
+    primaryContainer = Blue30,
+    onPrimaryContainer = Blue90,
+    inversePrimary = Blue40,
+    secondary = DarkBlue80,
+    onSecondary = DarkBlue20,
+    secondaryContainer = DarkBlue30,
+    onSecondaryContainer = DarkBlue90,
+    tertiary = Yellow80,
+    onTertiary = Yellow20,
+    tertiaryContainer = Yellow30,
+    onTertiaryContainer = Yellow90,
+    error = Red80,
+    onError = Red20,
+    errorContainer = Red30,
+    onErrorContainer = Red90,
+    background = Grey10,
+    onBackground = Grey90,
+    surface = Grey10,
+    onSurface = Grey80,
+    inverseSurface = Grey90,
+    inverseOnSurface = Grey20,
+    surfaceVariant = BlueGrey30,
+    onSurfaceVariant = BlueGrey80,
+    outline = BlueGrey60
 )
 
-private val LightColorPalette = lightColors(
-    primary = Purple500,
-    primaryVariant = Purple700,
-    secondary = Teal200
-
-    /* Other default colors to override
-    background = Color.White,
-    surface = Color.White,
+private val AndroidAcademyLightColorScheme = lightColorScheme(
+    primary = Blue40,
     onPrimary = Color.White,
-    onSecondary = Color.Black,
-    onBackground = Color.Black,
-    onSurface = Color.Black,
-    */
+    primaryContainer = Blue90,
+    onPrimaryContainer = Blue10,
+    inversePrimary = Blue80,
+    secondary = DarkBlue40,
+    onSecondary = Color.White,
+    secondaryContainer = DarkBlue90,
+    onSecondaryContainer = DarkBlue10,
+    tertiary = Yellow40,
+    onTertiary = Color.White,
+    tertiaryContainer = Yellow90,
+    onTertiaryContainer = Yellow10,
+    error = Red40,
+    onError = Color.White,
+    errorContainer = Red90,
+    onErrorContainer = Red10,
+    background = Grey99,
+    onBackground = Grey10,
+    surface = Grey99,
+    onSurface = Grey10,
+    inverseSurface = Grey20,
+    inverseOnSurface = Grey95,
+    surfaceVariant = BlueGrey90,
+    onSurfaceVariant = BlueGrey30,
+    outline = BlueGrey50
 )
 
 @Composable
 fun AndroidAcademyAppTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
```

Comment:
if `darkTheme` is s not used you can move it into function to prevent overrides

Reply:
Actualy I'm using it for previews. So i left it there